File: 2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.swift
class Solution {
	func mostWordsFound(_ sentences: [String]) -> Int {
		var result: Int = 0
		
		for sentence in sentences {
			var countSpace: Int = 1
			
			for cha in sentence {
				if (cha != " ") {
					continue
				}
				countSpace += 1
			}
			
			result = max(result, countSpace)
		}
		
		return result
	}
}

File: 2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.cs
public class Solution
    {
      public int MostWordsFound(string[] sentences)
      {
        int result = 0;
        foreach (var sentence in sentences)
        {
          int currentLength = sentence.Split(" ").Length;
          result = Math.Max(result, currentLength);
        }
        return result;
      }
    }